build/cli.version.darwin

```diff
@@ -0,0 +1 @@
+1.0.0.001540
```

Comment:
Double check this. I believe we want a different version on osx... likewise for linux.